plugins/scaffolder-backend/sample-templates/docs-template/{{cookiecutter.component_id}}/catalog-info.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   description: {{cookiecutter.description | jsonify}}
   annotations:
     github.com/project-slug: {{cookiecutter.storePath | jsonify}}
-    backstage.io/techdocs-ref: github:https://github.com/{{cookiecutter.storePath}}
+    backstage.io/techdocs-ref: url:https://github.com/{{cookiecutter.storePath}}/tree/master
```

Comment:
Can we do something about the hardcoded `master` ? Does scaffolder ask what the name of the default branch should be? If yes, we can add the cookiecutter variable here.

Reply:
Dont remember for sure and cant try it right now, but I think you can leave the `tree/master` part out and just point to the repo if you want the default branch.

Reply:
😅 `url:https://github.com/backstage/backstage` was my first try. But on the API response, it gives

```
InputError: GitHub URL must contain branch to be able to fetch tree at GithubUrlReader.readTree
```

However, that would really be a cool feature, to get the default branch if no tree is provided. cc @Rugvip Just in case you already have something on this. It might be in `// TODO` code comments. 😁

Reply:
https://github.com/backstage/backstage/blob/6e1497527797635ce65dfca3ec7a7d07411962ef/packages/backend-common/src/reading/GithubUrlReader.ts#L98

:)

Reply:
Are the repositories created with a master or a main branch by the scaffolder? Or is that even up to the scaffolder or is it just going with the github default?

Reply:
At least GitHub defaults to main now when creating a new repo. So maybe we should *at least* follow that?

Reply:
💯 Absolutely.

However, using a different branch name in `backstage.io/techdocs-ref` than what Scaffolder creates will break the docs workflow. Hence, maybe we should keep it as `/tree/master` for now?

But if Scaffolder can create a different branch name and provide `cookiecutter.branchName`, then we should replace `master` with it. @benjdlambert any comments?